doc/api/stream.md

```diff
@@ -2892,8 +2899,7 @@ the handling of backpressure and backpressure-related errors:
 
 ```js
 const { pipeline } = require('stream');
-const util = require('util');
-const fs = require('fs');
+const { pipeline: pipelinePromise } = require('stream/promises');
```

Comment:
fs is still required in this example for calling createWriteStream

```suggestion
const { pipeline: pipelinePromise } = require('stream/promises');
const fs = require('fs');
```

Reply:
fixed